Contribute/hacktoberfest.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Hacktoberfest and Microsoft docs
+description: Learn how to discover which docs repositories participate in Hacktoberfest, how to contribute, and what you can expect as a contributor.
+author: IEvangelist
+ms.author: dapine
+ms.date: 09/10/2021
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+---
+
+# Hacktoberfest and Microsoft docs
+
+Hacktoberfest is an annual world-wide event that drives open source contributions through the month of October. During the month, open source developers are encouraged to contribute to repositories through pull requests (PRs). There are many Microsoft docs open source repositories on GitHub, all of which contribute to the [docs.microsoft.com](https://docs.microsoft.com) content. Some of the repositories actively participate in the Hacktoberfest event. In this article you'll learn how to discover which repos are accepting pull requests, and what you can expect as a contributor.
+
+## Discoverability
+
+To discover if a docs repo is participating in Hacktoberfest, you will see the **hacktoberfest** topic on the project.
+
+:::image type="content" source="media/hacktoberfest/github-topic.png" lightbox="media/hacktoberfest/github-topic.png" alt-text="GitHub .NET docs repository with hacktoberfest topic.":::
+
+To filter all Microsoft docs and .NET repos that have the **hacktoberfest** topic see, [GitHub Topics: Hacktoberfest](https://github.com/topics/hacktoberfest?q=org%3AMicrosoftDocs+org%3Adotnet).
+
+Alternatively, a repository may choose to use the `Hacktoberfest` label instead. This is convenient for filtering issues with &mdash; for more information see, [Filtering issues and pull requests by labels](https://docs.github.com/github/administering-a-repository/finding-information-in-a-repository/filtering-issues-and-pull-requests-by-labels).

```

Comment:
Is the &mdash here intentional? It looks like this is a placeholder for a term that was to be inserted.

Reply:
```suggestion
Alternatively, a repository may choose to use the `Hacktoberfest` label instead. This is convenient for filtering issues. For more information see, [Filtering issues and pull requests by labels](https://docs.github.com/github/administering-a-repository/finding-information-in-a-repository/filtering-issues-and-pull-requests-by-labels).
```